src/app/models/post.go

```diff
@@ -21,21 +19,29 @@ type (
 		Updated    time.Time  `json:"updated" sql:"DEFAULT:current_timestamp"`
 	}
 
-	Search struct {
-		Word string `json:"word" validate:"required"`
-	}
-
 	Upload struct {
 		Path string `json:"path"`
 	}
 )
 
-func FindAllPost() []Post {
+func SearchPost(pages int, search string) []Post {
```

Comment:
__[gometalinter]__
* [staticcheck] other declaration of SearchPost
* [unconvert] other declaration of SearchPost
* [gosimple] other declaration of SearchPost

<!-- Comment by [SideCI](https://sideci.com) -->